src/compiler/typescript_to_rust/azle_generate/src/ts_ast/source_map/private_get_source_file_info.rs

```diff
@@ -0,0 +1,37 @@
+use swc_common::{Loc, SourceMap, Span};
+
+use crate::ts_ast::source_map::get_source_file_info::GetSourceFileInfo;
+
+pub(super) trait PrivateGetSourceFileInfo {
```

Comment:
Let's just not go here, I've only seen this type of visibility cause pain (in my limited experience)

Reply:
I will remove super.